theano/sandbox/rng_mrg.py

```diff
@@ -1456,10 +1456,13 @@ def choice(self, size=1, a=None, replace=True, p=None, ndim=None,
         Parameters
         ----------
         size: integer or integer tensor (default 1)
-            The number of samples. It should be between 1 and `p.shape[1]-1`
-        a: None
+            The number of samples. It should be between 1 and `p.shape[1]-1`.
+        a: None (default 2)
```

Comment:
The type specification doesn't make sense.  What comes after the parameter name is the type.  I think you want `int` instead of `None`.